.bazelrc.common

```diff
@@ -74,3 +60,7 @@ build:release --stamp --workspace_status_command=$(pwd)/workspace_status.sh
 # Config to enable building and testing with bzlmod
 common:bzlmod --enable_bzlmod
 common:bzlmod --@aspect_bazel_lib//lib:flag_bzlmod
+
+common --enable_platform_specific_config
+
+test:windows --test_tag_filters=-no-windows
```

Comment:
nit: could we just use `target_compatible_with` to exclude these tests?

Reply:
good call. updated.